terraform/20-app/outputs.tf

```diff
@@ -85,3 +85,9 @@ output "lambda" {
   }
 }
 
+output "kms_key_id" {
```

Comment:
Do we need this output as well? I can't see it being referenced anywhere

Reply:
No we don't so deleted - good catch.